### PR TITLE
COZMO-7707 Improve backpack and animation documentation

### PR DESCRIPTION
**Pull request by MarkWesleyAtAnki**
Documented what each of the 5 backpack lights refers to.
Added not on backpack lights regarding left/right being red LED only
Added set_center_backpack_lights function that forces left/right LEDs to off, which is the more likely use-case.
Added warning to play_anim regarding how we recommend play_anim_trigger instead as individual animations may be renamed or removed in future app versions.

### CODE REVIEW

File: src/cozmo/robot.py [hunk 1]
```diff
@@ -1028,6 +1052,10 @@ def play_anim(self, name, loop_count=1):
         if you wish to wait for completion (or listen for the
         :class:`cozmo.anim.EvtAnimationCompleted` event).
 
+        Warning: Specific animations may be renamed or removed in future updates
+            of the app. If you want your program to work more reliably across
+            all versions we recommed using :meth:`play_anim_trigger` instead.
```
Comment (msintov) [inline src/cozmo/robot.py]: Should be "recommend"

File: examples/tutorials/01_basics/07_backpack_lights.py [hunk 2]
```diff
@@ -35,6 +35,9 @@ def cozmo_program(robot: cozmo.robot.Robot):
     # set all of Cozmo's backpack lights to blue, and wait for 2 seconds
     robot.set_all_backpack_lights(cozmo.lights.blue_light)
     time.sleep(2)
+    # set just Cozmo's center backpack lights to white, and wait for 2 seconds
+    robot.set_center_backpack_lights(cozmo.lights.white_light)
```
Comment (gwatts) [inline examples/tutorials/01_basics/07_backpack_lights.py]: off topic but i wish i'd named all those light related constants in caps to be consistent with other constants.. brain fart on my part.

File: src/cozmo/robot.py [hunk 3]
```diff
@@ -938,8 +938,17 @@ def say_text(self, text, play_excited_animation=False, use_cozmo_voice=True, dur
...
+        Note: The left and right lights only contain red LEDs, so e.g. setting
+        them to green will look off, and setting them to white will look red
+
         Args:
-            light1-5 (class:'cozmo.lights.Light'): The lights for Cozmo's backpack.
+            light1 (class:'cozmo.lights.Light'): The left backpack light
+            light2 (class:'cozmo.lights.Light'): The front backpack light
+            light3 (class:'cozmo.lights.Light'): The center backpack light
+            light4 (class:'cozmo.lights.Light'): The rear backpack light
+            light5 (class:'cozmo.lights.Light'): The right backpack light
```
Comment (gwatts) [inline src/cozmo/robot.py]: should be backticks for the class references

Comment (gwatts) [inline src/cozmo/robot.py]: needs ASCII art for which light is which ;-)

File: src/cozmo/robot.py [hunk 4]
```diff
@@ -948,6 +957,21 @@ def set_backpack_lights(self, light1, light2, light3, light4, light5):
...
 
+    def set_center_backpack_lights(self, light):
+        '''Set the lights in the center of Cozmo's backpack to the same color.
+
+        Forces the lights on the left and right to off (this is useful as those
+        lights only support shades of red, so cannot generally be set to the
+        same color as the center lights).
+
+        Args:
+            light (class:'cozmo.lights.Light'): The lights for Cozmo's backpack.
```
Comment (gwatts) [inline src/cozmo/robot.py]: backticks

**Comment by MarkWesleyAtAnki** [comment]
Merged via a clean squashed branch in https://github.com/anki/cozmo-python-sdk/pull/63

**Pull request closed without merging**